### PR TITLE
Remove *_authenticated_developers_only (bad merge)

### PR DESCRIPTION
**Pull request by illusional**
Context: https://ci.hail.populationgenomics.org.au/batches/375915/jobs/40

### CODE REVIEW

File: ci/ci/ci.py [hunk 1]
```diff
@@ -20,9 +20,7 @@
     Database,
     check_csrf_token,
     monitor_endpoints_middleware,
-    rest_authenticated_developers_only,
     setup_aiohttp_session,
-    web_authenticated_developers_only,
```
Comment (lgruen) [inline ci/ci/ci.py:25]: Was this removal intentional (not included in the PR title)?

**Review by lgruen — APPROVED** [review]

File: ci/ci/ci.py (hunk 1 above)
Comment (illusional) [reply ci/ci/ci.py:25]: Yes, updated the title :)
